File: app/src/main/java/com/dhis2/usescases/datasets/datasetDetail/DataSetDetailComponent.java
package com.dhis2.usescases.datasets.datasetDetail;

import com.dhis2.data.dagger.PerActivity;

import dagger.Subcomponent;


@Subcomponent (modules = DataSetDetailModule.class)
@PerActivity
public interface DataSetDetailComponent {
    void inject(DataSetDetailActivity activity);
}

File: app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetDetail/DataSetDetailComponent.kt
package org.dhis2.usescases.datasets.dataSetTable.dataSetDetail

import dagger.Subcomponent
import org.dhis2.data.dagger.PerFragment

@Subcomponent(modules = [DataSetDetailModule::class])
@PerFragment
interface DataSetDetailComponent {
    fun inject(dataSetDetailFragment: DataSetDetailFragment)
}
